modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/services/export/TaskExportService.scala

```diff
@@ -26,18 +29,33 @@ class TaskExportService {
 
   private val tmpDir: File = new File("/tmp")
 
+  def responseFromTaskId(tasks: List[Task], isOneTask: Boolean, taskId: Int): (Long, InputStreamResource) = {
+    var file: File = null
+    if (isOneTask) {
+      file = exportTask(taskId)
+    } else {
+      file = exportTasks(tasks)
+    }
+    val contentLength = file.length()
+    (contentLength, new InputStreamResource(Files.newInputStream(file.toPath, StandardOpenOption.DELETE_ON_CLOSE)))
+  }
```

Comment:
Instead of passing a list or a TaskId here, you can simply pass a list with an element and always execute `exportTasks`.